es6.md

```diff
@@ -189,3 +189,35 @@ interface ImportNamespaceSpecifier {
 ```
 
 A namespace import specifier, e.g., `* as foo` in `import * as foo from "mod.js"`.
+
+## ExportDeclaration
+
+```js
+interface ExportDeclaration <: Node {
```

Comment:
@jeffmo I feel the same, that's why I wanted to have a more specific `declaration`, that could lead to a more strict constructor for the tree. I guess the grammar itself is the root of the evil here. I thought on splitting `ExportDeclaration` into `ExportDeclaration` and `ExportDefaultDeclaration`, but I guess that will not help that much, still we will have a lot of invalid states.